README.md

```diff
@@ -1,59 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
+> This website creates a working version of the leaderboard that preserves user input. The website is built using medium wireframe
 
-> One paragraph statement about the project.
+> |Mobile Version|Desktop Version
+> |--------------|--------------|
+> |![screenshot](Leaderboard-mobile.png).|![screenshot2](LeaderBoard-desktop.png)| 
```

Comment:
- [ ] The desktop version screenshot is not working, kindly fix this issue, make sure that you are using the right filename.
    - I would recommend moving the screenshots to another folder 😄

Reply:
Thank you for bringing my attention to this, I have corrected it and it displays properly. 

I have also made changes to the `MIT.md` file and filled the necessary spaces with my name and the name of this repository.

Thank you once again.